pkg/detectors/accuweather/accuweather.go

```diff
@@ -71,6 +74,24 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 	return results, nil
 }
 
+func verifyAccuweather(ctx context.Context, client *http.Client, resMatch string) (bool, error) {
+	req, err := http.NewRequestWithContext(ctx, "GET", "https://dataservice.accuweather.com/locations/v1/cities/autocomplete?apikey="+resMatch+"&q=----&language=en-us", nil)
+	if err != nil {
+		return false, err
+	}
+	res, err := client.Do(req)
+	if err != nil {
+		return false, err
+	}
+	defer res.Body.Close()
+	if res.StatusCode == http.StatusOK {
+		return true, nil
+	} else if res.StatusCode != http.StatusUnauthorized {
```

Comment:
Was this based on documentation or testing?

Reply:
original implementation